controller/src/main/java/com/esm/epam/controller/ExceptionsHandler.java

```diff
@@ -48,4 +52,24 @@
         ErrorResponse errorResponse = new ErrorResponse(5, exception.getLocalizedMessage());
         return new ResponseEntity<>(errorResponse, BAD_REQUEST);
     }
+
+    @ExceptionHandler(ServiceException.class)
```

Comment:
ExceptionHandler class should be in a separate package. Also, avoid naming like Exception**S**Handler.